sklearn/preprocessing/_encoders.py

```diff
@@ -145,10 +145,11 @@ class OneHotEncoder(_BaseEncoder):
 
     Parameters
     ----------
-    categories : 'auto' or a list of lists/arrays of values.
+    categories : 'auto' (default) or a list of lists/arrays of values.
```

Comment:
Reading and following other docstrings, I will write:

`categories: 'auto' or a list of list/array of values, default='auto' `

Reply:
I think we like to have ``default=auto`` at the end of the line, unless that makes the line too long.